packages/affiliates/libs/affiliates/utils.js

```diff
@@ -0,0 +1,323 @@
+// collection of functions to compose a automation pipeline with regards to dev/dapp mining.
```

Comment:
High-level comment: these function feel more contextual than just utils. Not saying we should change the structure, but I could imagine these being split out into more well-defined libraries or class-type objects that have a more refined purpose.

For instance, the config generation functions seem quite specialized and maybe more suited for a special file or class just dedicated to generating configs.

Reply:
I agree, though I dont think i want to do this for this PR.  This could be organized better ( though im not sure how yet), i'm using utils as a catch all for these loosely affiliated static functions.

Reply:
SG!